MapboxNavigation.podspec

```diff
@@ -44,7 +44,7 @@ Pod::Spec.new do |s|
   s.module_name = "MapboxNavigation"
 
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
-  s.dependency "Mapbox-iOS-SDK", "~> 5.0.0"
+  s.dependency "Mapbox-iOS-SDK", "~> 5.1"
```

Comment:
Also update MapboxNavigation-Documentation.podspec, Cartfile, Cartfile.resolved (via `carthage update`, and MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock.